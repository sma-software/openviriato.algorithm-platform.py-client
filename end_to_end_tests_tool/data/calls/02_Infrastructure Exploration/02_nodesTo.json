{
    "restResource": "/nodes",
    "httpMethod": "GET",
    "urlParameters": null,
    "queryParameters": [
        {
            "key": "HasSectionTrackToNodeID",
            "value": "161",
            "optional": true
        }
    ],
    "requestBody": null,
    "CSharpMethodName": "GetNodesWithSectionTrackTo(toNodeID)",
    "CSharpMethodParameterTypes": [ "System.Int64" ],
    "subcategory": "Get Neighboring Nodes"
}
