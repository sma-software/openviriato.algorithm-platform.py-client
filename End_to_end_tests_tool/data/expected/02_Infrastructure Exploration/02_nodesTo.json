[
  {
    "id": 348,
    "code": "85NH",
    "nodeTracks": [
      {
        "id": 349,
        "code": "1",
        "debugString": "stationtrack:85NH_1"
      },
      {
        "id": 350,
        "code": "2",
        "debugString": "stationtrack:85NH_2"
      }
    ],
    "debugString": "station:85NH"
  },
  {
    "id": 281,
    "code": "85JE",
    "nodeTracks": [],
    "debugString": "station:85JE"
  }
]